### PR TITLE
Fix/refactor/show selected diff

### PR DESCRIPTION
**Pull request by manhatsu**

### CODE REVIEW

**Comment by asana17** [comment]
Thank you for the small refactorings here: abc27bfba351aefc5556cdbc1516f35378d82153, that is very helpful. If you don't mind, could you split the commit to separate each change? It's tedious, but it can help people understand easily.

File: lua/super-commit/git/commands.lua [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+  M.show_output(bufnum, "git diff --cached ".. path)
 end
 
-local diff_keymap_msg = "To show git diff of files, select the file path in the window below, and press Enter."
-                        
-M.commands = {}
-M.commands[2] = "echo " .. diff_keymap_msg
-M.commands[3] = "git status"
-M.commands[4] = "git diff --name-only --cached"
+local file_sel_suggestion = "To show git diff of files, " ..
```
Comment (asana17) [inline lua/super-commit/git/commands.lua:15]: It is up to you, but the word `sel` is ambiguous.
```suggestion
local file_select_suggestion = "To show git diff of files, " ..
```

File: lua/super-commit/super-commit.lua
Comment (asana17) [inline lua/super-commit/super-commit.lua:1]: If you put `()`  after a function name, it makes clear that it is a function.
Also, it is better to write the scope of this change.
```
refactor(git/commands.lua): define callback outside set_autocmd()
```

**Review by asana17 — APPROVED** [review]
No major concerns.
You can merge the PR after fixing small comments.

File: lua/super-commit/git/commands.lua (hunk 1 above)
Comment (manhatsu) [reply lua/super-commit/git/commands.lua]: Changed variable name to `file_select_suggestion` in 788bd3aad0d2e8c32752314bf3200ea78e0ab325.

File: lua/super-commit/super-commit.lua
Comment (manhatsu) [reply lua/super-commit/super-commit.lua:1]: Changed commit message in f8945189a67bf5214ba8f9fa7dbd0da410dac17a.

File: lua/super-commit/git/commands.lua (hunk 1 above)
Comment (manhatsu) [reply lua/super-commit/git/commands.lua]: Split the commits to 76e8f034e7c6e3f62571a8f1258a2cc23081edf3 and 788bd3aad0d2e8c32752314bf3200ea78e0ab325.

**Pull request merged**